### ISSUE TITLE
Update skeleton examples with screen reader text

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5071]
**Describe the issue. What is the expected and unexpected behavior?**
I'm noticing that when I created the screenreaderText prop, I should have added it to all existing examples and shown the custom variations that could exist in our examples. 

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Enhancement for additional documentation
